tests/OpenCFP/Http/Controller/Admin/SpeakersControllerTest.php

```diff
@@ -11,7 +11,7 @@ class SpeakersControllerTest extends \PHPUnit_Framework_TestCase
 {
     public $app;
 
-    public function setup()
+    protected function setup()
```

Comment:
Method should also be named `setUp()`, but I like to keep both PRs and commits small, so happy to follow up with a separate PR.